src/Parsers/ASTSystemQuery.cpp

```diff
@@ -206,80 +219,87 @@ void ASTSystemQuery::formatImpl(const FormatSettings & settings, FormatState &,
     }
     else if (type == Type::SUSPEND)
     {
-        settings.ostr << (settings.hilite ? hilite_keyword : "") << " FOR "
-            << (settings.hilite ? hilite_none : "") << seconds
-            << (settings.hilite ? hilite_keyword : "") << " SECOND"
-            << (settings.hilite ? hilite_none : "");
+        print_keyword(" FOR ") << seconds;
+        print_keyword(" SECOND");
     }
     else if (type == Type::DROP_FORMAT_SCHEMA_CACHE)
     {
         if (!schema_cache_format.empty())
-            settings.ostr << (settings.hilite ? hilite_none : "") << " FOR " << backQuoteIfNeed(schema_cache_format);
+        {
+            print_keyword(" FOR ");
+            print_identifier(schema_cache_format);
+        }
     }
     else if (type == Type::DROP_FILESYSTEM_CACHE)
     {
         if (!filesystem_cache_name.empty())
         {
-            settings.ostr << (settings.hilite ? hilite_none : "") << " " << filesystem_cache_name;
+            settings.ostr << ' ';
+            print_identifier(filesystem_cache_name);
             if (!key_to_drop.empty())
             {
-                settings.ostr << (settings.hilite ? hilite_none : "") << " KEY " << key_to_drop;
+                print_keyword(" KEY ");
+                print_identifier(key_to_drop);
                 if (offset_to_drop.has_value())
-                    settings.ostr << (settings.hilite ? hilite_none : "") << " OFFSET " << offset_to_drop.value();
+                {
+                    print_keyword(" OFFSET ");
+                    settings.ostr << offset_to_drop.value();
+                }
             }
         }
     }
-    else if (type == Type::UNFREEZE)
+    else if (type == Type::DROP_SCHEMA_CACHE)
     {
-        settings.ostr << (settings.hilite ? hilite_identifier : "") << backQuoteIfNeed(backup_name);
```

Comment:
Here was a bug, it should be printed with `quoteString` and also `WITH NAME` missed.